src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -50,7 +50,7 @@ <h2 id="question-details"><b>Question {{ model.questionNumber }}: </b>{{ model.q
           </p>
         </div>
         <div class="form-check" *ngIf="hasSectionTeam">
-            <input type="checkbox" id="showSectionTeam" name="showSectionTeam" value="sectionTeam" (change)="toggleSectionTeam($event)">
+            <input type="checkbox" class="showSectionTeam" name="showSectionTeam" value="sectionTeam" (change)="toggleSectionTeam($event)">
             <label class="form-check-label ms-1" for="showSectionTeam">Show Section/Team</label>
```

Comment:
If I'm not wrong, the `for` attribute here won't work since the `input` no longer has `id` of `showSectionTeam`? Maybe can wrap the `label` around the `input`?

Reply:
Good catch! Will revert this change and make the IDs unique instead.